network/http_post/source/main.c

```diff
@@ -0,0 +1,182 @@
+#include <stdlib.h>
+#include <string.h>
+#include <stdio.h>
+#include <inttypes.h>
+
+#include <3ds.h>
+
+Result http_post(const char* url, const char* data)
+{
+	Result ret=0;
+	httpcContext context;
+	char *newurl=NULL;
+	u32 statuscode=0;
+	u32 contentsize=0, readsize=0, size=0;
+	u8 *buf, *lastbuf;
+
+	printf("POSTing %s\n", url);
+
+	do {
+		ret = httpcOpenContext(&context, HTTPC_METHOD_POST, url, 0);
+		printf("return from httpcOpenContext: %" PRIx32 "\n",ret);
+
+		// This disables SSL cert verification, so https:// will be usable
+		ret = httpcSetSSLOpt(&context, SSLCOPT_DisableVerify);
+		printf("return from httpcSetSSLOpt: %" PRIx32 "\n",ret);
+
+		// Enable Keep-Alive connections (on by default, pending ctrulib merge)
```

Comment:
And lose the end of this comment so it's just // Enable Keep-Alive connections

we're not pending a PR after all :p